rules/tacacs.dep

```diff
@@ -33,3 +33,15 @@ $(BASH_TACPLUS)_CACHE_MODE  := GIT_CONTENT_SHA
 $(BASH_TACPLUS)_DEP_FLAGS   := $(SONIC_COMMON_FLAGS_LIST)
 $(BASH_TACPLUS)_DEP_FILES   := $(DEP_FILES)
 
+
+
+
+SPATH       := $($(AUDISP_TACPLUS)_SRC_PATH)
+DEP_FILES   := $(SONIC_COMMON_FILES_LIST) rules/tacacs.mk rules/tacacs.dep
+DEP_FILES   += $(SONIC_COMMON_BASE_FILES_LIST)
+DEP_FILES   += $(shell git ls-files $(SPATH))
+
+$(AUDISP_TACPLUS)_CACHE_MODE  := GIT_CONTENT_SHA
+$(AUDISP_TACPLUS)_DEP_FLAGS   := $(SONIC_COMMON_FLAGS_LIST)
+$(AUDISP_TACPLUS)_DEP_FILES   := $(DEP_FILES)
```

Comment:
not sure why this change? can you create a separate pr for this one?

Reply:
I can, no problem. It is a simple fix for the cache of tacplus